helm/alert-logic/templates/daemonset.yaml

```diff
@@ -21,6 +21,8 @@ spec:
     spec:
       tolerations:
       - operator: "Exists"
+      - key: node-role.kubernetes.io/master
```

Comment:
Please also take out the introduced key & Effect, as only the "Exists" will suffice

Reply:
Basically the "Exists" operator says to deploy on all nodes, no matter the taint, even on masters.

Reply:
For reasons unknown, the official alert logic spec says not to install on masters. Any reason why they would do that?

Reply:
Not really. I would expect them to give a reason for that :)

Reply:
Cool, I've asked Alert Logic

Reply:
Actually, by looking at the config they recommend: 

`tolerations:
      - key: node-role.kubernetes.io/master
      - effect: NoSchedule
`
this translates in running the pods **only** on the master nodes.

Reply:
The response I got from Alert Logic was (I'd already asked before you corrected my misunderstanding). FYI

```
I believe this does not prevent scheduling on the master node; on the contrary, this has the effect of scheduling al-agent-container on the master node anyway, despite the master node being tainted with NoSchedule -- see https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/ . So without this clause, the agent container would not be scheduled on the master nodes by default, because the master is by default tainted with NoSchedule.
```

Reply:
Indeed, that's what I commented above :).
In the end I think we should keep the current toleration with operator: Exists, as this will schedule on all the nodes.

Reply:
Yes, thought you might like to see their response anyway.

I agree 👍